game/scripts/vscripts/components/duels/duels.lua

```diff
@@ -0,0 +1,309 @@
+-- Taken from bb template
+if Duels == nil then
+  DebugPrint ( 'Creating new Duels object.' )
+  Duels = class({})
+
+  Debug.EnabledModules['duels:*'] = true
+  Debug.EnabledModules['zonecontrol:*'] = true
```

Comment:
really these lines should just be removed once the module works well. it uses zonecontrol a lot so i left that one enabled.